internal/core/config.go

```diff
@@ -146,6 +147,7 @@ type SecurityChecksWorkflowConfig struct {
 // commonWorkflowConfigOpts holds common configuration options that are applicable for all
 // workflows.
 type commonWorkflowConfigOpts struct {
+	Assignees   []string `yaml:"assignees"`
```

Comment:
Do not add `assignees` field here as it will get added to all structs that use `commonWorkflowConfigOpts`. Add it to `Global` struct inside `GithubWorkflowConfiguration{}`.